src/gan/gen_test.py

```diff
@@ -0,0 +1,39 @@
+import argparse
```

Comment:
Based on the name here, what's going on with this? Is it just a short-hand test file or what? I guess what I'm asking is, how does this fit into the overall _production_ system or is it separate from it? If separate, then I'd suggest creating a folder called `tests` in `src/gan` and put this file there.